composer/packages/theme/src/definitions/views/api-designer.less

```diff
@@ -29,22 +29,68 @@
             .action-container {
                 margin-bottom: 1em;
             }
-
-            .accordion.ui.fluid.styled {
-                padding-bottom: .5em !important;
+            .accordion {
                 margin-top: 20px;
                 border-left: 0;
                 border-right: 0;
                 border-top: 1px solid #e8e8e8;
                 border-bottom: 1px solid #e8e8e8;
-                .inline-editor {
-                    font-weight: 500;
-                    &:hover {
-                        opacity: 0.5;
+                box-shadow: none;
+                .path-op-container {
+                    display: inline;
+                    float: right;
+                    .get {
+                        border: 1px solid #2980b9;
+                        color: #2980b9;
+                        font-weight: 400;
```

Comment:
Move common styles to a separate class and inherit

Reply:
Will be tracked with issue #13334